webpack/assets/javascripts/react_app/components/Layout/Navigation.js

```diff
@@ -71,15 +78,22 @@ const Navigation = ({
             } else {
               groups[currIndex].groupItems.push({
                 ...sub,
-                isActive: currentPath === sub.href?.split('?')[0],
+                isActive:
+                  currentPath === sub.href?.split('?')[0] ||
+                  sub.title === currentLocation ||
```

Comment:
Can this only check for loc/org if the item is a loc/org item? in case someone names them with the same name as other menu items(like "facts"), which is unlikely but better to be safe

Reply:
For sure, I get what you meant now with checking the classname